pkg/pillar/cmd/zedmanager/updatestatus.go

```diff
@@ -412,17 +434,37 @@ func doActivate(ctx *zedmanagerContext, uuidStr string,
 		need := uint64(config.FixedResources.Memory) << 10
 		if remaining < need {
 			var errStr string
+			retryable := false
```

Comment:
That is misleading; it will retry when some application halts. It is merely the case that there isn't sufficient applications currently in halting state, so the time it will be retried is less predictable.

Reply:
Ah, ok, sorry, missed the logic here, will modify

Reply:
I fixed the logic, thank you for pointing me